profiler/test/Datadog.Profiler.IntegrationTests/ApplicationInfoTest.cs

```diff
@@ -63,17 +62,19 @@ private static (string ServiceName, string Environment, string Version) ExtractS
                 text = reader.ReadToEnd();
             }
 
-            return (
-                ServiceName: ExtractTag("service", text),
-                Environment: ExtractTag("env", text),
-                Version: ExtractTag("version", text));
-        }
+            var match_tags = Regex.Match(text, "\"tags_profiler\":\"(?<tags>[^\"]*)\"", RegexOptions.Compiled);
```

Comment:
Would it be possible to show an example of the expected format as a comment?
Easier to read than regex  :^)

Reply:
good point :) will do